src/step07.rb

```diff
@@ -9,18 +9,22 @@ module Step07
 
   def self.create_dialog
     html_file = File.join(__dir__, 'html', 'step07.html')
-    width = 15 + 128 + 15 + 15 + 128 + 15 # Margins and contents
-    # Add extra space for older SU versions that doesn't support control of
-    # inner size.
-    width += 20 if Sketchup.version.to_f < 21.1
+    # Define pixel perfect dimensions matching our content.
+    width = 15 + 128 + 15 + 15 + 128 + 15
+    height = 460
+    # Add some extra space for older SSU versions that don't support inner sizing.
```

Comment:
"SSU" => "SU"

Reply:
You have a problem with Sveriges socialdemokratiska ungfomsförbund?

(Fixed it)